core/src/main/java/org/elasticsearch/plugins/InstallPluginCommand.java

```diff
@@ -237,6 +240,10 @@ private Path download(Terminal terminal, String pluginId, Path tmpDir) throws Ex
         }
 
         // fall back to plain old URL
+        if (pluginId.contains("://") == false) {
```

Comment:
I used to use file:/path/to/file.zip instead of file://
Not sure if it respects standards but it works.